specification/nginx/resource-manager/NGINX.NGINXPLUS/preview/2025-03-01-preview/swagger.json

```diff
@@ -0,0 +1,2850 @@
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v6/types.json#/definitions/systemData",
+          "readOnly": true
+        }
+      }
+    },
+    "NginxDeploymentWafPolicyApplyingStatus": {
+      "type": "object",
+      "title": "Nginx Deployment Waf Policy Applying Status",
+      "readOnly": true,
+      "properties": {
+        "code": {
+          "$ref": "#/definitions/NginxDeploymentWafPolicyApplyingStatusCode"
+        },
+        "displayStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A readable string of the current status, and sometimes have the reason for the current state."
+        },
+        "time": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The date and time in UTC the current applying status was set."
+        }
+      }
+    },
+    "NginxDeploymentWafPolicyApplyingStatusCode": {
+      "type": "string",
+      "example": "Applying",
+      "readOnly": true,
+      "description": "Machine readable code indicating the applying status code of a WAF Policy.",
+      "enum": [
+        "NotApplied",
+        "Applying",
+        "Succeeded",
+        "Failed",
+        "Removing"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "NginxDeploymentWafPolicyApplyingStatusCode",
+        "values": [
+          {
+            "value": "NotApplied",
+            "description": "The policy is not referenced in the nginx config and not applied."
+          },
+          {
+            "value": "Applying",
+            "description": "The policy is referenced in the nginx config and is applying."
+          },
+          {
+            "value": "Succeeded",
+            "description": "The policy is referenced in the nginx config and that config has been successfully applied."
+          },
+          {
+            "value": "Failed",
+            "description": "The policy is referenced in the nginx config and that config failed to apply."
+          },
+          {
+            "value": "Removing",
+            "description": "The policy is now not referenced in the nginx config and its being removed from the applied nginx config."
+          }
+        ]
+      }
+    },
+    "NginxDeploymentWafPolicyCompilingStatus": {
+      "type": "object",
+      "title": "Nginx Deployment Waf Policy Compiling Status",
+      "readOnly": true,
+      "properties": {
+        "code": {
+          "$ref": "#/definitions/NginxDeploymentWafPolicyCompilingStatusCode"
+        },
+        "displayStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A readable string of the current status, and sometimes have the reason for the current state. If the CompilingStatus is Failed the Display Status will be The waf Policy failed to compile."
+        },
+        "time": {
```

Comment:
>time [](http://example.com/codeflow?start=9&length=4)

you could come up with a clearer name than 'time', like 'startedCompilingAt' or 'compiledAt'

Reply:
Since this was already merged in the private repo pr - https://github.com/Azure/azure-rest-api-specs-pr/pull/21186
Can we consider this in future API versions ?